src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13012,6 +13031,9 @@ void VRFTerminalUnitEquipment::CalcVRF_FluidTCtrl(EnergyPlusData &state,
                       _,
                       _,
                       state.dataHVACVarRefFlow->MaxHeatingCapacity(VRFCond));
+            // yujie: update TU air mass flow
+            state.dataLoopNodes->Node(state.dataHVACVarRefFlow->VRFTU(VRFTUNum).VRFTUInletNodeNum).MassFlowRate =
+                state.dataDXCoils->DXCoil(this->HeatCoilIndex).InletAirMassFlowRate;
```

Comment:
This is not standard practice to update another model's operating conditions. Why doesn't the VRF model know what air flow is being used?

Reply:
In the original code, the air mass flow of the coil is updated here through `FanSpdRatio`.
![image](https://github.com/NREL/EnergyPlus/assets/13421161/fb5d9588-fc69-4d76-881e-e4130b22f0f1)
As a result, the rate of air flowing into the coil should be updated as well. This is added in this PR, in the original code, the `thisDXCoil.InletAirMassFlowRate` is not adjusted when `FanSpdRatio` changes.
![image](https://github.com/NREL/EnergyPlus/assets/13421161/149dc461-4ba9-4ae4-bf88-0b3c968daa0b)
The node corresponding to the inlet of the terminal unit containing that coil should probably be updated as well (is this right?). That's the reason for this change.

Reply:
I think I would delete the new lines at 13034- 13046 and then follow suggestion below.